integrations/tomcat/prometheus_metadata.yaml

```
@@ -0,0 +1,40 @@
+platforms:
+  - type: GKE
+    launch_stage: HIDDEN
+    detections:
+      - characteristic_metric:
+          metric_type: prometheus.googleapis.com/tomcat_bytessent_total/counter
+    exporter_metadata:
+      name: Tomcat Prometheus Exporter
+      doc_url: https://github.com/bitnami/containers/tree/main/bitnami/jmx-exporter
+      minimum_supported_version: "0.17.0"
+    default_metrics:
+      - name: prometheus.googleapis.com/tomcat_bytessent_total/counter
+        prometheus_name: tomcat_bytessent_total
+        kind: CUMULATIVE
+        value_type: DOUBLE
+      - name: prometheus.googleapis.com/tomcat_errorcount_total/counter
+        prometheus_name: mysql_up
```

this needs to be tomcat_errorcount_total
```suggestion
        prometheus_name: tomcat_errorcount_total
```